doc/nrf/app_dev/device_guides/nrf54h/ug_nrf54h20_gs.rst

```diff
@@ -217,10 +229,11 @@ To prepare the nRF54H20 DK for first use, you must manually program the values o
 Programming the nRF54H20 SoC binaries
 =====================================
 
-After programming the BICR, you need to provide the nRF54H20 SoC with the provisioning of the nRF54H20 SoC binaries, a bundle containing the precompiled firmware for the Secure Domain and System Controller.
+After programming the BICR, program the nRF54H20 SoC with the nRF54H20 SoC binaries.
+This bundle contains the precompiled firmware for the Secure Domain and System Controller.
```

Comment:
```suggestion
This bundle contains the precompiled firmware for the :ref:`Secure Domain <ug_nrf54h20_secure_domain>` and :ref:`System Controller <ug_nrf54h20_sys_ctrl>`.
```
May be good to link to Secure Domain and System Controller sections?

Reply:
I'd link to the ABI compatibility page, containing the soc binaries